packages/core-logger-winston/__tests__/logger.test.ts

```diff
@@ -1,16 +1,23 @@
 import { AbstractLogger } from "@arkecosystem/core-logger";
 import * as capcon from "capture-console";
 import "jest-extended";
+import * as winston from "winston";
 import { WinstonLogger } from "../src";
 
 let logger: AbstractLogger;
 let message;
 
 beforeAll(() => {
     const driver = new WinstonLogger({
+        level: "debug",
         transports: [
             {
                 constructor: "Console",
+                package: "@arkecosystem/core-logger-winston/__tests__/__fixtures__/transport",
```

Comment:
There is no need to use a fixture for that, simply use the winston console transport.

Reply:
thought I already changed it, guess I forgot to push changes! doing so now